### ISSUE TITLE
"Unable to parse HTTP body" when doing "GET"

### ISSUE THREAD

**Issue by JHereu** [balderdashy/sails-postgresql#166]
I have some problems with the `sails-postgresql` adapter. I was using `sails-mongo` in a Sails project but now I want to try Postgre. It shouldn't be problematic cause changing the adapter in `/config/models.js` has to work. 

The adapter creates the tables (`migrate: alter`) and till there, it's fine. Also, Sails lifts without errors. But when it comes to do a `get` request from whichever model, it throws the following error:

    Unable to parse HTTP body- error occurred :: Error (E_UNKNOWN) :: Encountered an unexpected error
    error: invalid input syntax for integer: "NaN"
        at Connection.parseE (<myProjectPath>/node_modules/sails-postgresql/node_modules/pg/lib/connection.js:534:11)
    at Connection.parseMessage (<myProjectPath>/node_modules/sails-postgresql/node_modules/pg/lib/connection.js:361:17)
    at Socket.<anonymous> (<myProjectPath>/node_modules/sails-postgresql/node_modules/pg/lib/connection.js:105:22)
    at Socket.emit (events.js:107:17)
    at readableAddChunk (_stream_readable.js:163:16)
    at Socket.Readable.push (_stream_readable.js:126:10)
    at TCP.onread (net.js:538:20)

    Details:  error: invalid input syntax for integer: "NaN"

Furthermore, I'm using SQL Developer to consult the data of my postgre database, but it throws an error too when try to open any table:

    ERROR: Column "a" does not exist

But I have no column named "a" anywhere! Nevertheless, after the error, the table is shown. When I consult them through command line, it doesn't have any error neither.

It's kind of annoying cause it's not the first time it occurs, so i'd be glad if you throw some light on that matter...

Thanks.

**Comment by sailsbot** [comment]
Thanks for posting, @JHereu.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by nahtnam** [comment]
Were you able to come up with a solution? I have the same problem here.

**Comment by nahtnam** [comment]
Fixed it! Was making the request to `/model/new?asdf=asdf` instead of `/model/create?asdf=asdf`.
